server/sonar-web/src/main/js/apps/coding-rules/rule/custom-rule-creation-view.js

```diff
@@ -128,7 +128,7 @@ export default ModalFormView.extend({
         value
       };
     }).get();
-    options.params = params.map(function (param) {
+    options.params = params.map(param => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)